gps/deduce.go

```diff
@@ -167,18 +168,49 @@ func (m bitbucketDeducer) deduceRoot(path string) (string, error) {
 	return "bitbucket.org" + v[2], nil
 }
 
+func getBitbucketRepo(s string) (string, error) {
```

Comment:
Worth posting a doc string that has a sample input and output?

Reply:
I was planning on writing a test for this, since it makes sure it works and accomplishes the same.